tests/Support/SupportStrTest.php

```diff
@@ -0,0 +1,303 @@
+<?php
+
+namespace Tightenco\Collect\Tests\Support;
+
+use Tightenco\Collect\Support\Str;
+use PHPUnit\Framework\TestCase;
+
+class SupportStrTest extends TestCase
```

Comment:
Is this the same deal? Pulled in the Str dependency but failed to pull in its test?